packages/forms/src/Components/BelongsToSelect.php

```diff
@@ -199,9 +199,9 @@ public function getOptionLabelFromRecord(Model $record): string
         return $this->evaluate($this->getOptionLabelFromRecordUsing, ['record' => $record]);
     }
 
-    public function getDisplayColumnName(): string
```

Comment:
Breaking change?

Reply:
Nah, no-one should be calling this method